nox/sessions.py

```diff
@@ -379,6 +379,10 @@ def run(
         if not args:
             raise ValueError("At least one argument required to run().")
 
+        if len(args) == 1 and isinstance(args[0], list):
+            raise ValueError("First argument to `session.run` is a list. "
+                               "Did you mean to use `session.run(*args)`?")
```

Comment:
```suggestion
            msg = "First argument to `session.run` is a list. Did you mean to use `session.run(*args)`?"
            raise ValueError(msg)
```

Reply:
Sure! Is the error message still OK as-is? That's why I originally removed tuple, because I didn't want to get into the weeds

Reply:
I thought of that, thought it probably was okay to leave. You could use Sequence if you wanted to be a little more general - "first argument to `session.run` is a Sequence. Did you mean...

Reply:
I accepted your suggestion as-is. Happy to make another update, if you prefer.